docs/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Erratum advisory name to flaw filter (OSIDB-922)
 - CORS allow-list functionality (OSIDB-967, OSIDB-965)
 
+### Changed
+- Flaw creation and update triggers taskman (OSIDB-861)
```

Comment:
I am not sure the users would know what `taskman` is :smile: so I would rather replace it with eg. `Jira task sync`